mcp/server/tools/src/services/ProjectService.ts

```
@@ -0,0 +1,110 @@
+import { Service } from 'typedi';
+import { makeApiCall } from '../commons/utils/axios.js';
+import { CONFIGS } from '../commons/configs/index.js';
+
+@Service()
+export default class ProjectService {
+    constructor(
+    ){}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)